recipe/meta.yaml

```
@@ -34,9 +34,14 @@ source:
     sha256: d7864460ce8def8e0620cb9dd891e94299ade75eac75aa96b34001af3cdb074c  # [win]
 
 build:
-  number: 0
+  number: 1
   skip: true  # [osx]
 
+requirements:
+  run_constrained:
+    # Prevent clobbering with cccl
+    - cccl <0a0
```

In [thrust PR](https://github.com/conda-forge/thrust-feedstock/pull/18/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR27) we used this syntax:
```cuda-cccl <0.0.0a0```
Should we follow that? I'm honestly not too familiar with this syntax.

---

Shouldn't matter as far as I am aware. This is what @jakirkham suggested here: https://github.com/conda-forge/staged-recipes/pull/23722#discussion_r1298815269

---

I decided to match the previous syntax.
```suggestion
    - cccl <0.0.0a0
```